governance_blueprint/validation/validate_artifacts.py

```
@@ -301,6 +344,71 @@ def validate_report_structure() -> list[str]:
+            "icgc compute governance",
+        }
+        normalized_seen = {value.casefold() for value in seen_frameworks}
+        missing_frameworks = sorted(expected_frameworks.difference(normalized_seen))
+        if missing_frameworks:
+            errors.append(
+                f"Regulatory playbook CSV missing required framework mappings: {missing_frameworks}"
+            )
+    return errors
+
+
+def validate_master_reference_markdown() -> list[str]:
```

Pylint (4.0.5)

[refactor] Either all return statements in a function should return an expression, or none of them should.

(R1710)